particles/Tutorial/Javascript/Demo/source/RandomComputer.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/* global defineParticle */
+
+defineParticle(({SimpleParticle, log}) => {
+
+  function getMove(b) {
```

Comment:
Something seems unfinished here ... ? getMove is unrolled below.

Reply:
Oops, meant to delete the function.